reviews/graduation-prometheus.md

```diff
@@ -0,0 +1,60 @@
+# Prometheus Graduation Application
+
+Prometheus was the second accepted project into the CNCF (joined in May 2016) and has grown significantly over time. In August 2017 we have successfully hosted a community conference (PromCon) in collaboration with the CNCF that attracted 200+ attendees from the developer and user community.
+
+The following application links to the required information to become a graduated project.
+
+## Prometheus fulfills all the incubating and graduation criteria:
+
+### Document that it is being used successfully in production by at least three independent end users which, in the TOC’s judgement, are of adequate quality and scope.
+
+* "Users" section of https://prometheus.io/
+* In-progress PR to add an `ADOPTERS.md` file: https://github.com/prometheus/prometheus/pull/3833/files
```

Comment:
It's "USERS.md", but this is a nitpick.

Reply:
The governance criteria and @caniszczyk suggested an `ADOPTERS.md`, so maybe it's best to update that in the linked PR instead.

Reply:
@caniszczyk suggested both, but it seems ADOPTERS is the canonical place. @brian-brazil 's point that this specific repo might not be the best place remains, but I think we have consensus that it's fine to put somewhere.

Reply:
I'd be for the docs repo (like the governance) unless we want to have per-repo user lists, but let's discuss that somewhere else :)

Reply:
Isn't the users section on prometheus.io sufficient? In the long run, we probably want a separate section, as it will grow too much for the landing page, but we can keep it there indefinitely.

Reply:
@beorn7 Going to comment on https://github.com/prometheus/docs/pull/969 about that.